docs/getting_started/install_ez_platform.md

```diff
@@ -150,7 +150,9 @@ run the following command:
 
 It is recommended to add your project to version control.
 
-For git, you can do it in the following way:
+First, create a `.gitignore` file based on the `.gitignore.dist` provided in the project.
```

Comment:
Does it mean that we no longer assume people could be using other VCSs?
If the assumption still stands, we could begin the sentence with something like:
"If you use GIT,...

Reply:
We don't actually, considering the code is hosted on git.